src/ops/view.rs

```diff
@@ -13,6 +14,7 @@ pub(super) fn pretty_view(
     summaries: &[Summary],
     owners: &Option<Vec<String>>,
     suggest_cargo_tree_command: bool,
+    config: &Config,
```

Comment:
What about we only pass the current work dir here? I am not sure if it makes sense to pass the whole config to `pretty_view`. It seems we can pass the minimal information to it. (I am not sure if this really helps anything, so just want to discuss and I don't have any strong options about it)

Reply:
Its pretty common practice within cargo itself to just pass `Config` everywhere.